src/Components/Aspire.Seq/SeqSettings.cs

```diff
@@ -19,7 +21,26 @@ public sealed class SeqSettings
     public string? ApiKey { get; set; }
 
     /// <summary>
-    /// Gets or sets the base URL of the Seq server (including protocol and port). E.g. "https://example.seq.com:6789"
+    /// Gets or sets the base URL of the Seq server (including protocol and port). E.g. "https://example.seq.com:6789. Overrides endpoints set on <c>Logs</c> and <c>Traces</c>."
     /// </summary>
     public string? ServerUrl { get; set; }
+
+    /// <summary>
+    /// Gets OTLP exporter options for logs.
+    /// </summary>
+    public OtlpExporterOptions Logs
+    {
+        get;
+        private set;
+    } = new ();
```

Comment:
```suggestion
    public OtlpExporterOptions Logs { get; } = new();
```

No need for a private setter, it is never used.

Reply:
done